auto_install/install.sh

```diff
@@ -1055,10 +1055,11 @@ confOVPN() {
     # verify server name to strengthen security
     $SUDO sed -i "s/SRVRNAME/${SERVER_NAME}/" /etc/openvpn/easy-rsa/pki/Default.txt
 
-    if [ ! -d "/home/$pivpnUser/ovpns" ]; then
-        $SUDO mkdir "/home/$pivpnUser/ovpns"
+    INSTALL_HOME=$(cat /etc/passwd | grep "$INSTALL_USER" | cut -d: -f6)
+	if [ ! -d "$INSTALL_HOME/ovpns" ]; then
+        $SUDO mkdir "$INSTALL_HOME/ovpns"
     fi
-    $SUDO chmod 0777 -R "/home/$pivpnUser/ovpns"
+    $SUDO chmod 0777 -R "$INSTALL_HOME/ovpns"
```

Comment:
this one might have slipped me for some reason, not your fault since it was already here but i wonder Why 777 here? 
this should never be the case! 
worst-case scenario 755 for dirs 744 for files (umask 022)

@orazioedoardo  @Giraffe1966 you have any clue?

Reply:
It's very old [apparently](https://github.com/pivpn/pivpn/commit/53565dd4fe86d1b7b2f65636b1d32004a38199f7#diff-299d9b30c952310aace4aba77c698874R479). However, since `pivpn -a` removes permssion from other when creating a file and `/home/pi` is 755 other users on the system won't be able to read them.

Reply:
No wait, it's still bad, other users on the system can read file in the `ovpns` folder but they can delete them `rwx` on `others`.
I think the line should be `$SUDO chown "$INSTALL_USER" "$INSTALL_HOME/ovpns"` Folders are 755 by default.

Reply:
well .. let's leave this out of the scope for this PR ... ill fix it myself before testing the test branch a merge it .. planning to do it next weekend.

Reply:
> well .. let's leave this out of the scope for this PR

Agree

> ill fix it myself before testing the test branch and merge it

Ok, remember that there is another one in the makeOVPN.sh.

Reply:
ill just do a search through all scripts for 777 perms and fix all i find